assignments/prototypes.js

```diff
@@ -32,6 +58,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(attrs){
+  GameObject.call(this, attrs);
+  CharacterStats.call(this, attrs);
```

Comment:
don't need to the `GameObject` constructor here. `CharacterStats` is already inheriting from `GameObject`, so Humanoid should be able to inherit both `CharacterStats` and `GameObject` all from `CharacterStats`